packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -460,10 +460,12 @@ class _ExpansionPanelListState extends State<ExpansionPanelList> {
               : null,
         ),
       );
-      expandIconContainer = !child.canTapOnHeader
-          ? MergeSemantics(
+      if (!child.canTapOnHeader) {
```

Comment:
You could always wrap the expandIconContainer above in semantics and set the container property to !child.canTapOnHeader

Reply:
(If you do that, you could even go back to not having the expandIconContainer and just have it all defined inline below)

Reply:
Since we are using Semantics with label now, I worry that wrapping it regardless will messed up the label if we merge semantics at the end